packages/init/package.json

```diff
@@ -1,71 +1,91 @@
-    "test": "jest",
-    "tsc": "tsc --project tsconfig.json --noEmit",
-    "watch": "tsc --project tsconfig.json --noEmit --watch"
-  },
-  "dependencies": {
-    "@hapi/hapi": "^20.2.0",
-    "@prismicio/types-internal": "^1.5.3",
-    "@slicemachine/client": "1.1.17",
-    "@slicemachine/core": "1.1.17",
-    "adm-zip": "^0.5.9",
-    "analytics-node": "^6.0.0",
-    "axios": "^0.24.0",
-    "chalk": "^4.1.2",
-    "fp-ts": "^2.11.5",
-    "fs-extra": "^10.0.0",
-    "inquirer": "^8.2.0",
-    "io-ts": "^2.2.16",
-    "open": "^8.2.1",
-    "ora": "^5.4.1",
-    "tmp": "^0.2.1",
-    "uuid": "^8.3.2"
-  },
-  "devDependencies": {
-    "@rollup/plugin-replace": "^3.0.1",
-    "@types/adm-zip": "^0.4.34",
-    "@types/analytics-node": "^3.1.7",
-    "@types/fs-extra": "^9.0.13",
-    "@types/hapi__hapi": "^20.0.9",
-    "@types/inquirer": "^8.1.3",
-    "@types/jest": "^27.0.2",
-    "@types/mock-fs": "^4.13.1",
-    "@types/tmp": "^0.2.2",
-    "@types/uuid": "^8.3.4",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "better-npm-audit": "^3.4.0",
-    "depcheck": "^1.4.3",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-import": "^2.24.0",
-    "eslint-plugin-jest": "^24.4.0",
-    "jest": "^29.3.1",
-    "mock-fs": "^5.1.2",
-    "nock": "^13.1.3",
-    "siroc": "^0.16.0",
-    "stdout-stderr": "^0.1.13",
-    "ts-jest": "^29.0.3",
-    "typescript": "^4.4.3"
-  },
-  "engines": {
-    "node": ">=14"
-  },
-  "gitHead": "e34025f35abbd621d3bcdae3772114e3436f381e"
+	"name": "@slicemachine/init",
+	"version": "0.1.1-dev-plugins.11",
+	"description": "Init Prismic Slice Machine in your project",
+	"keywords": [
+		"typescript",
+		"prismic"
+	],
+	"repository": {
+		"type": "git",
+		"url": "ssh://git@github.com/prismicio/slice-machine.git",
+		"directory": "packages/init"
+	},
+	"license": "Apache-2.0",
+	"author": "Prismic <contact@prismic.io> (https://prismic.io)",
+	"type": "module",
+	"exports": {
+		".": {
+			"require": "./dist/index.cjs",
+			"import": "./dist/index.js"
+		},
+		"./cli": {
+			"require": "./dist/cli.cjs",
+			"import": "./dist/cli.js"
+		},
+		"./package.json": "./package.json"
+	},
+	"main": "dist/index.cjs",
+	"module": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"bin": {
+		"slicemachine-init": "./bin/slicemachine-init.js"
+	},
+	"files": [
+		"bin",
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"build": "vite build",
+		"dev": "vite build --watch",
+		"format": "prettier --write .",
+		"lint": "eslint --ext .js,.ts .",
+		"prepare": "npm run build",
+		"size": "size-limit",
+		"test": "npm run lint && npm run unit && npm run build && npm run size",
+		"unit": "vitest run --coverage",
+		"unit:watch": "vitest watch",
+		"depcheck": "depcheck --config=.depcheckrc"
+	},
+	"dependencies": {
+		"@antfu/ni": "^0.18.8",
+		"@lihbr/listr-update-renderer": "^0.5.3",
```

Comment:
It's a fork of `listr-update-renderer` to adjust the output format. I recommend we move the code into `@slicemachine/init`'s package so that it can be easily traced and updated.

cc @lihbr